frontend/src/providers/ContextProvider.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Meta Platforms, Inc. and affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable react-refresh/only-export-components */
```

Comment:
Are you sure you need to disable these? 

it's worth the effort to remove "any" and have explicit types. Strong typing will help early error discovery.